semantic-core/src/Data/Core.hs

```diff
@@ -61,18 +69,19 @@ data Core f a
   | String Text
   -- | Load the specified file (by path).
   | Load (f a)
-  | Edge Edge (f a)
```

Comment:
The removal of `Edge` means that edges (whether imports or lexical) are discovered either via instrumentation or by encoding.